src/app/(home-page)/home/page.jsx

```diff
@@ -0,0 +1,36 @@
+import AiIndex from 'components/pages/home/ai-index';
+import Bento from 'components/pages/home/bento';
+import GetStarted from 'components/pages/home/get-started';
+import Hero from 'components/pages/home/hero/hero';
+import Industry from 'components/pages/home/industry';
+import InstantProvisioning from 'components/pages/home/instant-provisioning';
+import Lightning from 'components/pages/home/lightning';
+import Logos from 'components/pages/home/logos';
+import Multitenancy from 'components/pages/home/multitenancy';
+import Trusted from 'components/pages/home/trusted';
+import SEO_DATA from 'constants/seo-data';
+import getMetadata from 'utils/get-metadata';
+
+export const metadata = getMetadata({
+  ...SEO_DATA.index,
```

Comment:
would you add `rel="canonical"`?

Reply:
That tag is already added automatically. (see https://neon.tech/home src)

Reply:
indeed!